es6-destructuring/main.js

```diff
@@ -0,0 +1,53 @@
+// destructuring object 1
+
+let book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+let { title, author, libraryID } = book1;
+console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+// destructuring object 2
+
+let book2 = {
+  title: "Les Fleurs du mal",
+  author: "Charles Baudelaire",
+  libraryID: 2345
+};
+let { title: title2, author: author2, libraryID: libraryID2 } = book2;
+console.log(`The title of the book is ${title2}, the author is ${author2}, and the library id is ${libraryID2}`)
+
+// destructuring array
+
+const library = [
+  {
+    title: 'The Road Ahead',
+    author: 'Bill Gates',
+    libraryID: 1254
+  },
+  {
+    title: 'Walter Isaacson',
+    author: 'Steve Jobs',
+    libraryID: 4264
+  },
+  {
+    title: 'Mockingjay: The Final Book of The Hunger Games',
+    author: 'Suzanne Collins',
+    libraryID: 3245
+  },
+  {
+    title: "Dead Dead Demon's De De De De Destruction",
+    author: 'Inio Asano',
+    libraryID: 1233
+  }
+];
+
+function getBooks() {
+  return [library[0], library[1], library[2]];
+}
+
+let [book3, book4, book5] = getBooks();
+console.log(book3);
+console.log(book4);
+console.log(book5);
```

Comment:
It seems like you've missed the last couple steps.

---

![image](https://user-images.githubusercontent.com/16143771/127590383-ba7abe63-1a68-48fe-b5ae-eaa5c6c8802e.png)

Reply:
es6-destructuring - Fernando Figueiredo
![Screen Shot 2021-07-29 at 10 28 44 PM](https://user-images.githubusercontent.com/24905313/127600324-1d8be76e-3ad8-4a68-adc3-8e32e061c7ec.png)